app/scripts/components/404.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 import React from 'react';
 import { connect } from 'react-redux';
+import createReactClass from 'create-react-class';
 
-var NotFound = React.createClass({
+var NotFound = createReactClass({
```

Comment:
I was going to suggest that if we're going to bite of all the work of upgrading anyway, we might as well move from this style of declaring classes to use the new ES6 syntax. But that will be even more changes that have to happen at every component. It would probably be nice to do at some point, but perhaps we don't have to do it now